arcaflow_plugin_stressng/stressng_schema.py

```diff
@@ -842,9 +826,9 @@ class CommonOutput:
     bogo_ops_per_second_usr_sys_time: typing.Annotated[
         float,
         schema.id("bogo-ops-per-second-usr-sys-time"),
-        schema.name("Bogus operations per second per user and sys time"),
+        schema.name("Bogus operations per second in user and sys time"),
         schema.description(
-            "is the bogo-ops rate divided by the user + system time."
+            "The bogo-ops rate divided by the user + system time. "
```

Comment:
A _rate_ divided by an amount of time yields an average _acceleration_....  That should be, "The number of bogo-ops", not the rate.  Alternatively, it could be, "The bogo-ops rate as a function of CPU time".